### PR TITLE
Update en.json added Public Folder Local strings and added general strings for mirror, toggle, and local

### PR DESCRIPTION
**Pull request by RueShejn**

### CODE REVIEW

**Comment by Frooxius** [comment]
Hmm... I'm not 100 % sure about the Mirror, Local and Toggle being general strings that are split up. What are these for exactly? It seems like they're a bit more specific things for the mirror, so we shouldn't put those into general, because the way some of these are translated can depend on the context. 

E.g. we could add a string "Local Mirror" or "Local Toggle" specifically.

**Comment by shadowpanther** [comment]
Yes, please don't split your strings to single words. Adjectives are gendered in some languages, so *Local* Mirror and *Local* Toggle would translate "Local" differently.

File: en.json [hunk 1]
```diff
@@ -1366,8 +1369,10 @@
         "Tutorial.Intro.BadgesDisabilitySpeech" : "Speech Impaired",
         "Tutorial.Intro.BadgesTitle" : "Badges",
         "Tutorial.Intro.BadgesDescription" : "Badges are small icons which show aspects of a user. Here are a few important badges to remember. The disability badges can be self assigned though the NeosBot.",
-        "Tutorial.Intro.HelpSite" : "Help",
-       
+        "Tutorial.Intro.HelpSite" : "Help
```
Comment (shadowpanther) [inline en.json:1372]:
```suggestion
        "Tutorial.Intro.HelpSite" : "Help",
```
You lost a quote and a comma there

**Comment by Frooxius** [comment]
Closing this one, we're doing the changes fresh due to some merge conflicts.

**Pull request closed without merging**